package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "pod-dependency-init-container",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

```suggestion
  "version": "1.0.2",
```
can you please change this. `1.0.1` is already in tags, I forgot to update the same in package.json thanks